targets/TARGET_GigaDevice/mbed_rtx.h

```diff
@@ -0,0 +1,41 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *

```

Comment:
Can you add SPDX identifier to new files (like HAL implementation , this config file).

```
/*
 * Copyright (c) 2009-2018 Arm Limited. All rights reserved.
 *
 * SPDX-License-Identifier: Apache-2.0
 *
 * Licensed under the Apache License, Version 2.0 (the License); you may
```

Reply:
OK, we will add the SPDX identifier to the new files and there are a few questions that need to be confirmed.

1. Files in \targets\TARGET_GigaDevice\TARGET_GD32F30X\GD32F30x_standard_peripheral are our standard peripheral library. The license of these files will be changed to the following, could you please help us confirm this change?

/*
    Copyright (c) 2018, GigaDevice Semiconductor Inc. **All rights reserved.**

    **SPDX-License-Identifier: BSD-3-Clause**

    Redistribution and use in source and binary forms, with or without modification, 

2. The license of other files that follow Apache-2.0 licence will be changed to the following, and it is needed to be confirmed too.

/* 
 * Copyright (c) **2009-**2018 GigaDevice Semiconductor Inc. **All rights reserved.**
 *
 * **SPDX-License-Identifier: Apache-2.0**
 * 
 * Licensed under the Apache License, Version 2.0 (the "License");

Reply:
I was requesting files that contain Mbed OS porting code (like HAL us ticker, rtc and similar), not changing your SDK (peripheral library). We should use SPDX identifiers where we can. 

1st: dont need to change your peripheral library
2nd: files that contain Mbed OS porting should contain SPDX identifier

Does this clear it or ?

Reply:
Thank you for your detailed reply. Maybe we have referenced to other files and found other manufacturers' files haven't changed. Therefore, it is not clear how to add the SPDX identifier for us. And we will add a new line for "SPDX-License-Identifier: Apache-2.0" in the head of the licence declaration.